testScript.js

```diff
@@ -0,0 +1,105 @@
+const fs = require("fs");
```

Comment:
Do we need this file ?

Reply:
No, 
we donot need this
removing the file now.